backend/molgenis-emx2-sql/src/main/java/org/molgenis/emx2/sql/SqlDatabase.java

```diff
@@ -27,7 +27,7 @@ public class SqlDatabase extends HasSettings<Database> implements Database {
   public static final String ANONYMOUS = "anonymous";
   public static final String USER = "user";
   public static final String WITH = "with {} = {} ";
-  public static final int TEN_SECONDS = 10;
+  public static final int HUNDRED_SECONDS = 100;
```

Comment:
why would 100 seconds be needed , that would bring down the service in prod scenario

Reply:
indeed, will undo